src/partitionmanager.cpp

```diff
@@ -344,6 +345,16 @@ void PartitionManagerPrivate::unmount(const Partition &partition)
     }
 }
 
+void PartitionManagerPrivate::setLabel(const Partition &partition, const QString &label)
+{
+    qCInfo(lcMemoryCardLog) << "Can setLabel:" << externalMedia.match(partition.deviceName()).hasMatch() << partition.devicePath();
+    if (externalMedia.match(partition.deviceName()).hasMatch()) {
+        m_udisksMonitor->setLabel(partition.devicePath(), label);
+    } else {
+        qCWarning(lcMemoryCardLog) << "setLabel allowed only for external memory cards," << partition.devicePath() << "is not allowed";
```

Comment:
I would start this with capital s (i.e. `SetLabel allowed only...`) like it's done in the methods above (`mount()` -> `Mount`, `unmount()` -> `Unmount`) or alternatively would say `Renaming allowed only...` similar to how `format()` below has.